CRM/Event/Form/SelfSvcTransfer.php

```diff
@@ -247,8 +247,6 @@ public static function checkProfileComplete($fields, &$errors, $self) {
     }
     if (!$email && !(CRM_Utils_Array::value('first_name', $fields) &&
       CRM_Utils_Array::value('last_name', $fields))) {
-      $defaults = $params = ['id' => $eventId];
```

Comment:
Why is this removed?

Reply:
Unnecessary lines. `$eventId` is an undeclared variable.